packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/components/FlipbookControls.js

```diff
@@ -0,0 +1,151 @@
+import { Button, Box, Grid } from 'grommet'
+import { FormNext, FormPrevious } from 'grommet-icons'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled, { withTheme } from 'styled-components'
+import { useTranslation } from 'react-i18next'
+
+import locationValidator from '../../../helpers/locationValidator'
+
+const StyledButton = styled(Button)`
+  & > div {
+    flex-direction: column;
+  }
+`
+
+const backgrounds = { dark: 'dark-3', light: 'neutral-6' }
+
+const FlipbookControls = ({
+  currentFrame = 0,
+  locations = [],
+  onFrameChange = () => true,
+  theme
+}) => {
+  const { t } = useTranslation('components')
+
+  const handleKeyDown = (event) => {
+    switch (event.key) {
+      case 'ArrowRight': {
+        event.preventDefault()
+        event.stopPropagation()
+        handleNext()
```

Comment:
Do we want the sequence to loop when you reach the end, if you hold the key down?